test/grpc/server/server.go

```diff
@@ -0,0 +1,84 @@
+package server
```

Comment:
The directory / package of this file do no match the pattern used for `test/http2/server.go`.